frontend/pokino-app/src/app/api/api.config.ts

```diff
@@ -35,18 +35,25 @@ export class ApiConfig {
     }
 
     /**
-     * example: http://localhost:8001/game/clickReady/{playerId}
+     * example: http://localhost:8001/game/clickReady?playerName={playerName}&playerId={id}
      */
-    public static getPlayerReadyUrl(playerName: string): URL {
+    public static getPlayerReadyUrl(playerName: string, playerId: string): URL {
         const root: Array<string> = [this.ROOT_URL, this.GAME_PORT];
-        const endpoints: Array<string> = [this.GAME_ENDPOINT, this.READY_ENDPOINT, playerName];
-        return this.buildUrl(root, endpoints);
+        const endpoints: Array<string> = [this.GAME_ENDPOINT, this.READY_ENDPOINT];
+        let url = this.buildUrl(root, endpoints).href;
+        url += ('?playerName=' + playerName + '&playerId=' + playerId);
+        return new URL(url);
     }
 
+    /**
+     * example: http://localhost:8001/game/login?playerName={playerName}
```

Comment:
ist nicht ganz konsistent hier, weil ich die query param name und nicht playerName genannt habe (wie hier in dem beispiel korrekt wäre) - werde ich dann in einem anderen PR mal anpassen, oder aber du kannst es selber im login endpoint im backend endpoint anpassen wenn du willst - up to you